mappings/net/minecraft/block/Block.mapping

```diff
@@ -1,11 +1,11 @@
 CLASS none/akw net/minecraft/block/Block
 	FIELD A stateFactory Lnone/ask;
-	FIELD a ID Lnone/kp;
+	FIELD a DEFAULT_ID Lnone/kp;
 	FIELD b creativeTab Lnone/adq;
 	FIELD c defaultState Lnone/asj;
 	FIELD d unlocalizedName Ljava/lang/String;
 	FIELD h REGISTRY Lnone/cs;
-	FIELD i BLOCKSTATELIST Lnone/cx;
+	FIELD i BLOCKSTATE_IDLIST Lnone/cx;
```

Comment:
The field is an IDLIST, so I named it accordingly. Instead of just BLOCKSTATE_LIST.